GitHub-Actions/two-stage-pipeline-template/{{cookiecutter.outputDir}}/.github/workflows/pipeline.yaml

```diff
@@ -247,6 +324,16 @@ jobs:
           role-session-name: prod-deployment
           role-duration-seconds: 3600
           role-skip-session-tagging: true
+        
+      - name: Assume the prod pipeline user role via OIDC
+        if: env.PERMISSIONS_PROVIDER == 'OpenID Connect'
```

Comment:
I assume there's no straightforward way to get `OpenID Connect` into something more terminal-friendly? (i.e. no spaces)

Reply:
It could be changed to OIDC, but then it would just go straight to OIDC without ever having shown the user that it means OIDC if they only ran the init command and not bootstrap.

This is what the prompt looks like right now

Select a user permissions provider.
        1 - IAM
        2 - OpenID Connect
Choice []:

Reply:
In that case I'd lean toward `OpenID Connect (OIDC)` in the prompt.

I assume `click` doesn't provide some straightforward way to map the prompt choices to code-friendly names?

Reply:
Yea click doesn't really let you change the value that an option maps to, its basically what gets displayed is whats returned as the value. I worked around that in the bootstrap cli by just having dictionaries that mapped the display names to values. In this case since the questions are all loaded from the question.json file, if we change the prompt to `OpenID Connect (OIDC)` then the value in the pipeline files would have to match that exactly. If we want to have the value returned be different we'd need to add an extra JSON field and have Choice Questions try to map the answer to a value based on whats in that JSON.

Code for Choice(Question) for reference:
https://github.com/aws/aws-sam-cli/blob/d5bd26eea3500e80c3b25685f93422fa63157637/samcli/lib/cookiecutter/question.py#L235

Reply:
Purely from a customer's perspective, if there's no extra context before the prompts and the first OIDC-related mention is `OpenID Connect`, then `OpenID Connect (OIDC)` would be clearer, as OIDC is the term I'd expect people to use most commonly.

From a technical perspective there's not a huge difference; they're both not great as constants. But it shouldn't be visible in the customer pipeline template anyway, so it's probably fine.